src/doctypes.js

```diff
@@ -337,7 +337,7 @@ export const makeBalanceTransactionsConn = () => {
 export const appsConn = {
   query: () => Q(APP_DOCTYPE),
   as: 'apps',
-  fetchPolicy: older30s
+  fetchPolicy: neverReload
```

Comment:
The `transactionsConn` query stills have a `older30s` policy, whereas this is probalby the most impactful query?

Reply:
`transactionsConn` has a lot of `includes`. I'm not sure if Cozy-Client will refresh correctly datas when there is includes in fact. 

`transactionsConn` is used by: 
- DebugRecurrencePage
- Recurrence
- And reimboursement. 

Need to understand what is this "reimboursement" stuff to know if this query is really used when you navigate